drivers/hmis/app/models/hmis/workflow_execution/engine.rb

```diff
@@ -72,24 +72,26 @@ def next_task_steps(step)
       nodes.map { |node| steps_by_node_id[node.id] }.compact
     end
 
-    def send_message(...)
+    def send_message(user, ...)
       message = Hmis::WorkflowExecution::Message.new(...)
-      message_handler.call(message)
+      message_handler.call(message, user)
```

Comment:
Is there any reason `user` shouldn't be added to the message struct itself? https://github.com/greenriver/hmis-warehouse/blob/64ac6d5448462016ffd9970a543f017f72b47b3f/drivers/hmis/app/models/hmis/workflow_execution/message.rb#L5-L8

seems reasonable enough to add the user who submitted the step